volume/store/store.go

```diff
@@ -194,12 +194,14 @@ func (s *VolumeStore) create(name, driverName string, opts map[string]string) (v
 		}
 	}
 
-	logrus.Debugf("Registering new volume reference: driver %q, name %q", driverName, name)
```

Comment:
Well, my thought was that we're *requesting* a driver called `""`, which resulted in loading the `local` driver; I was wondering if we're interested in that information *before* the driver is actually loaded/resolved as well